bigchaindb/core.py

```diff
@@ -136,11 +138,54 @@ def write_transaction(self, signed_transaction, durability='soft'):
         signed_transaction = deepcopy(signed_transaction)
         # update the transaction
         signed_transaction.update({'assignee': assignee})
+        signed_transaction.update({'assignment_timestamp': time()})
 
         # write to the backlog
         response = r.table('backlog').insert(signed_transaction, durability=durability).run(self.conn)
         return response
 
+    def reassign_transaction(self, transaction, durability='hard'):
```

Comment:
We can do this, but it means a single node cluster (which is already timing out) will spend a lot of effort reassigning transactions to itself.